docs/cookbook/promotions/remove-promotion-action.rst

```diff
@@ -0,0 +1,59 @@
+How to remove a predefined promotion action?
+============================================
+
+Some promotion can be inappropriate to some business. Here is method to remove promotion actions in a compiler pass.
+
+Add a compiler pass into your project
+-------------------------------------
+
+In your kernel project class, you can add a custom compiler pass.
+
+.. code-block:: php
+
+    <?php
+
+    namespace App;
+
+    use App\DependencyInjection\Compiler\PromotionPass;
+
+    final class Kernel extends BaseKernel
+    {
+    // ...
+
+        protected function build(ContainerBuilder $container): void
+        {
+            $container->addCompilerPass(new PromotionPass());
+        }
+    }
+
+Remove project rule
+-------------------
+
+.. code-block:: php
+
+    <?php
+
+    namespace App\DependencyInjection\Compiler;
+
+    use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+    use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+    class PromotionPass implements CompilerPassInterface
+    {
+        public function process(ContainerBuilder $container): void
+        {
+            $parameter = $container->getParameter('sylius.promotion_actions');
+
+            $actionsToRemove = [
+                'sylius.promotion_action.shipping_percentage_discount'
```

Comment:
Missing coma at the end of line

Reply:
Maybe we could suggest removing promotions actions by their **type**? Then we could have a simple array like:

```
$actionsToRemove = [
    'shipping_percentage_discount',
];
```

And there would be no need to get the definition from the container :)

Reply:
This also refers to the next file

Reply:
Thx for your review. I amend modifications to my commit.